### DISCUSSION TITLE
Dreame App  - dont want to use the Xiaomi home app

### DISCUSSION THREAD

**Discussion by Sweti934** [Tasshack/dreame-vacuum#272] [Q&A] [answered]
Hi all,

I use the Xiamo home app for my 1st vaccum and air cleaner. So far so good. Now we got the L20 Ultra (We love it) for the ground floor. I  used their app as it is more detailed and has cooler features. Am I right, that I have to use this L20 ultra with the xiaomi home app in order to get it working in HA? Or is this also possible with the Dreame app?

thanks for any help!

**Comment by 13robin37** [comment, marked as answer]
Dreame Accounts are supported in the 2.0 beta, make sure to activate beta releases in HACS and you could use that. Then it's possible to pick a Dreame account during integration setup.
